autopub/publish_release.py

```diff
@@ -1,22 +1,17 @@
+import glob
 import os
 import sys
 
 sys.path.append(os.path.dirname(__file__))  # noqa
 
-from base import BUILD_SYSTEM, PYPI_URL, run_process
-
-poetry_pub = ["poetry", "publish", "-u", "$PYPI_USERNAME", "-p", "$PYPI_PASSWORD"]
```

Comment:
This no longer supports custom poetry repos (not sure if it did before). But since the repository for autopub is configurable I don't think there is much lost.

Reply:
I don't believe custom package repositories via `PYPI_URL` were ever supported for Poetry.